src/projects/api_server/controllers/v1/vhosts/apps/streams/streams_controller.cpp

```diff
@@ -94,9 +94,14 @@ namespace api
 						{
 							properties->SetUnusedStreamDeletionTimeout(jv_properties["unusedStreamDeletionTimeoutMs"].asInt());
 						}
+
+						if (jv_properties["defaultRtpCalculationMethod"].isNull() == false && jv_properties["defaultRtpCalculationMethod"].isInt())
+						{
+							properties->SetdefaultRtpCalculationMethod(jv_properties["defaultRtpCalculationMethod"].asInt());
```

Comment:
If your API receives these values as 1 or 2, your users will be confused. Rather, it would be better to use something like IgnoreRtcpSRTimestamp in boolean form. I would like the default to be false.